lib/js/jsre.nim

```diff
@@ -1,21 +1,9 @@
 ## Regular Expressions for the JavaScript target.
 ## * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
-
-runnableExamples:
-  let jsregex: RegExp = newRegExp(r"\s+", r"i")
-  jsregex.compile(r"\w+", r"i")
-  assert jsregex.test(r"nim javascript")
-  assert jsregex.exec(r"nim javascript") == @["nim".cstring]
-  assert jsregex.toString() == r"/\w+/i"
-  jsregex.compile(r"[0-9]", r"i")
-  assert jsregex.test(r"0123456789abcd")
-  assert $jsregex == "/[0-9]/i"
-  assert jsregex.len == 5
-
 when not defined(js):
   {.error: "This module only works on the JavaScript platform".}
 
-type RegExp* {.importjs.} = object    ## Regular Expressions for JavaScript target.
+type RegExp* {.importjs.} = ref object of JsRoot ## Regular Expressions for JavaScript target.
   flags* {.importjs.}: cstring        ## cstring that contains the flags of the RegExp object.
```

Comment:
please double check: are these `{.importjs.}` needed given that type itself is importjs?